source/_integrations/siren.markdown

```diff
@@ -0,0 +1,59 @@
+---
+title: Siren
+description: Instructions on how to set up siren devices within Home Assistant.
+ha_category:
+  - Siren
+ha_release: '2021.5'
+ha_domain: siren
+ha_quality_scale: internal
+ha_codeowners:
+  - '@home-assistant/core'
+  - '@raman325'
+ha_iot_class:
+---
+
+The `siren` integration is built for the controlling and monitoring of siren/chime devices.
```

Comment:
```suggestion
The Siren integration is built for the controlling and monitoring of siren/chime devices.
```

Additionally, we should clarify on this page this is used by other integrations and cannot be used by itself?

Reply:
I can add that, but I tried looking at the other platform docs and didn't see any of them mention that. Maybe I didn't find the right one? I looked at media_player, humidifier, and vacuum.

Reply:
Some have recently added information around that. Either way, I think it is important to make it clear to avoid questions.